### PR TITLE
remove common code and use dependency

### PR DESCRIPTION
**Pull request by PaulBredl**
## Description of changes
see https://github.com/IT-REX-Platform/template-microservice/pull/21

  
## How has this been tested?
All tests still run green, no functional changes were implemented.
    
## Checklist before requesting a review
- [x] My code follows the [coding guidelines](https://github.com/IT-REX-Platform/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My code fulfilles all acceptance criteria
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have made corresponding changes to the documentation
- [ ] I have added explanation of architectural decision and rationales to [wiki/adr](https://github.com/IT-REX-Platform/wiki/tree/main/adr)
- [ ] I have updated the changes in the ticket description

## Checklist for reviewer
- [ ] The code works and does not throw errors
- [ ] The code is easy to understand and there are no confusing parts
- [ ] The code follows the [coding guidelines](https://github.com/IT-REX-Platform/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of this project
- [ ] The code change accomplishes what it is supposed to do
- [ ] I cannot think of any use case in which the code does not behave as intended
- [ ] The added and existing tests reasonably cover the code change
- [ ] I cannot think of any test cases, input or edge cases that should be tested in addition
- [ ] Description of the change is included in the documentation

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.1-bin.zip
```
Comment (PaulBredl) [inline gradle/wrapper/gradle-wrapper.properties:3]: I don't know why these changes to the wrapper occur, maybe I use a newer gradle version than someone else

File: src/test/java/de/unistuttgart/iste/gits/courseservice/integration/MutationCreateChapterTest.java [hunk 2]
```diff
@@ -16,7 +16,7 @@
 import static org.hamcrest.Matchers.is;
 
 @GraphQlApiTest
-public class MutationCreateChapterTest {
+class MutationCreateChapterTest {
```
Comment (PaulBredl) [inline src/test/java/de/unistuttgart/iste/gits/courseservice/integration/MutationCreateChapterTest.java:19]: sonar issues fixed

File: src/test/resources/application.properties [hunk 3]
```diff
@@ -0,0 +1,10 @@
+server.port=2001
```
Comment (PaulBredl) [inline src/test/resources/application.properties:1]: we now need explicit test properties, as the test db url can no longer be hard coded in the common code

File: build.gradle [hunk 4]
```diff
@@ -3,6 +3,7 @@ plugins {
 	id 'org.springframework.boot' version '3.0.6'
 	id 'io.spring.dependency-management' version '1.1.0'
 	id "io.github.kobylynskyi.graphql.codegen" version "5.7.2"
+    id 'com.adarshr.test-logger' version '3.2.0'
```
Comment (PaulBredl) [inline build.gradle:6]: not required, but produces nicer test output

**Review by myluki2000 — APPROVED** [review]
lgtm
